src/sentry/incidents/grouptype.py

```diff
@@ -45,6 +52,79 @@ class MetricIssueEvidenceData(EvidenceData[float]):
 MetricResult = int | dict
 
 
+class SessionsAggregate(StrEnum):
+    CRASH_FREE_SESSIONS = "percentage(sessions_crashed, sessions) AS _crash_rate_alert_aggregate"
+    CRASH_FREE_USERS = "percentage(users_crashed, users) AS _crash_rate_alert_aggregate"
+
+
+ALERT_TYPE_IDENTIFIERS: dict[Dataset, dict[str, str]] = {
```

Comment:
For the typing `dict[Dataset, dict[str, str]]` is there a better type we can use or make? a str enum or type literal might be nice.